### ISSUE TITLE
uninitialized constant Paperclip::Storage::S3::AWS

### ISSUE THREAD

**Issue by bolom** [spree/spree#8205]
## Context
After i have upgraded Spree to 3.3. I have got this error 

<img width="1099" alt="capture d ecran 2017-08-03 a 18 25 26" src="https://user-images.githubusercontent.com/72155/28946523-29ae96fc-7879-11e7-9061-d3f22973c21b.png">

## Your Environment

* Version used:

gem 'spree', '~> 3.3.0.rc1'

* Gemfile and Gemfile.lock as text in a Gist:

https://gist.github.com/bolom/77baab14057dc67f97434a15077dfafb

**Comment by damianlegawiec** [comment]
Hi @bolom , with 3.3 we've bumped paperclip to version 5 which required `aws-sdk` >= 2, information about it should be in release notes and upgrade guide - will prepare this today! Thank you for reporting!

**Comment by bolom** [comment]
thanks!

**Comment by bolom** [comment]
I have updated my gem file

```
gem "paperclip", "~> 5.0.0"

gem 'spree', '~> 3.3.0.rc3'
gem 'spree_auth_devise', '~> 3.3'
gem 'spree_gateway', '~> 3.3'
gem 'spree_editor', github: 'spree-contrib/spree_editor'

gem 'spree_i18n',  github: 'spree-contrib/spree_i18n'
gem 'spree_sitemap', github: 'spree-contrib/spree_sitemap'
gem 'spree_braintree_vzero', github: 'spree-contrib/spree_braintree_vzero'
gem 'spree_mail_settings',github: 'spree-contrib/spree_mail_settings'

```

I have got this error

```
Bundler could not find compatible versions for gem "paperclip":
  In Gemfile:
    paperclip (~> 5.0.0)

    spree_i18n was resolved to 3.3.0.rc1, which depends on
      spree_core (< 4.0, >= 3.1.0) was resolved to 3.3.0.rc3, which depends on
        paperclip (~> 5.1.0)
```

spree_i18n doesn't with the latest version of Spree?

**Comment by damianlegawiec** [comment]
@bolom spree uses `paperclip` `~> 5.1.0`

this upgrade guide covers it all:

https://github.com/spree/spree/blob/master/guides/content/developer/upgrades/three-dot-two-to-three-dot-three.md#update-aws-sdk-gem-to--20
